api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -531,6 +531,12 @@
      */
     Attribute<String> LIBRARY_JARS = new Attribute<>(String2String.getInstance());
 
+    /**
+     * This configuration file can be used to configure ssl parameters over-riding the default Yarn ssl-server.xml
```

Comment:
Since this is any custom config, can you adjust the javadoc to the same. You can add additional documentation that it can be useful to set ssl parameters etc.